crates/testing/tests/unit/message.rs

```diff
@@ -46,17 +43,16 @@ fn version_number_at_start_of_serialization() {
         _pd: PhantomData,
     };
     let message = Message {
-        version,
         sender,
         kind: MessageKind::Consensus(SequencingMessage(Left(
             GeneralConsensusMessage::ViewSyncCommitCertificate(simple_certificate),
         ))),
     };
-    let serialized_message: Vec<u8> = bincode_opts().serialize(&message).unwrap();
+    let serialized_message: Vec<u8> = Serializer::<MAJOR, MINOR>::serialize(&message).unwrap();
     // The versions we've read from the message
-    let major_version_read = u16::from_le_bytes(serialized_message[..2].try_into().unwrap());
-    let minor_version_read = u16::from_le_bytes(serialized_message[2..4].try_into().unwrap());
 
-    assert_eq!(version.major, major_version_read);
-    assert_eq!(version.minor, minor_version_read);
+    let version_read = Version::deserialize(&serialized_message).unwrap().0;
```

Comment:
I think this actually changes the test -- it's now testing a property of the `Serializer` (round-trip version de/serialize = identity), where before it was testing that the binary representation actually had the version in the right place.

I think the previous test was good to have somewhere, though you probably already have these tests in `vbs`. I think we could just delete this file?

Reply:
Yes, there's tests in vbs... This probably should be deleted.